packages/cli-tool/package.json

```diff
@@ -23,6 +23,7 @@
     "@oclif/core": "^1",
     "@oclif/plugin-help": "^5",
     "@oclif/plugin-plugins": "^2.0.1",
+    "bootstrap": "^5.1.3",
```

Comment:
Do we need to add `bootstrap` in this package?

Reply:
🙈 I thought I removed it already >< 
Of course no, we should not have it there. It was inserted when the `npm install bootstrap` command was run in the wrong "cwd" (current working directory for the command to run).

Reply:
✅ 56167e0